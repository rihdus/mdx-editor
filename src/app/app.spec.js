import React from "react";
import { render, screen } from "@testing-library/react";
import App from './index'

test("app renders", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});