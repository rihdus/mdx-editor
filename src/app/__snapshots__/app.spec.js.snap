// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders 1`] = `
<div>
  <div
    style="outline: none;"
    tabindex="-1"
  >
    <div
      class="common_default_appBody__e5x7fm0"
    >
      <div
        class="common_default_explorerView__e5x7fm1"
      >
        <pre>
          MDX content goes here
        </pre>
        <h1 />
      </div>
    </div>
  </div>
</div>
`;
